typings/global.d.ts

```diff
@@ -101,6 +99,8 @@ declare module 'vue' {
     ElWatermark: typeof import('element-plus')['ElWatermark']
     ElTour: typeof import('element-plus')['ElTour']
     ElTourStep: typeof import('element-plus')['ElTourStep']
+    ElAnchor: typeof import('element-plus')['ElAnchor']
+    ElAnchorLink: typeof import('element-plus')['ElAnchorLink']
```

Comment:
Why have these two lines of code been moved here?

Reply:
I used git mv to rename the `typings/components.d.ts` file to `typings/global.d.ts`, but for some reason, github is comparing it with the `global.d.ts` file in the parent directory.

Reply:
Perhaps we can adjust the order of the ElAnchor and ElAnchor (L102,L103 -> L9,L10)

Reply:
ok, I’ve updated it.